blueocean-dashboard/src/main/js/creation/bitbucket/server/BbAddServerDialog.jsx

```diff
@@ -0,0 +1,155 @@
+import React, { PropTypes } from 'react';
+import { observer } from 'mobx-react';
+import debounce from 'lodash.debounce';
+import { Dialog, FormElement, TextInput } from '@jenkins-cd/design-language';
+
+import ServerErrorRenderer from './ServerErrorRenderer';
+
+
+let t = null;
```

Comment:
@i386 t is set to [props.flowManager.translate](https://github.com/jenkinsci/blueocean-plugin/blob/e495e01c5e0cba2ce61987174eb931be987202e0/blueocean-dashboard/src/main/js/creation/bitbucket/server/BbAddServerDialog.jsx#L25-L25), so it should be fine. No import needed.